src/utils/hooks/useRouter.tsx

```diff
@@ -34,6 +35,17 @@ interface RouterResult {
   location: any
 }
 
```

Comment:
Can you add a comment explaining what it does, its parameters and its return shape?

Reply:
This is just a wrapper for all the default available properties when using these:
```
  const params = useParams()
  const location = useLocation()
  const history = useHistory()
  const match = useRouteMatch()
```

I don't see why we need to document this one, which can be quite extensive. It is simply wrapping around used the Router individual hooks.

Reply:
I meant just for the hyphenKeys function, not the whole hook.

Reply:
Oh! Ok. Is just that you commented is flagging just a empty line in this file